memory.js

```diff
@@ -0,0 +1,24 @@
+$(function() {
+	var lastClicked;
+
+	$('td').click(function(){
+		if (lastClicked === undefined){
```

Comment:
Unless you need to check for `undefined` specifically, I would just simplify this to `if (!lastClicked)`.